R/finalSteps.R

```diff
@@ -55,9 +56,10 @@ toolREMINDdemand <- function(regrdemand, EDGE2teESmap, REMINDtall, SSP_scen) {
 toolCreateOutput <- function(logit_params, pref_data, ptab4W, vot_data, NEC_data,
                              capcost4W, demByTech, int_dat, intensity, capCost,
                              price_nonmot, complexValues, load_Factor, annual_mileage,
-                             demISO, SSP_scen, EDGE_scenario, level2path, output_folder) {
+                             demISO, SSP_scen, DEM_scen, EDGE_scenario, level2path, output_folder) {
   price_component <- MJ_km <- NULL
   gdp_scenario <- paste0("gdp_", SSP_scen)
+  dem_scenario <- paste0("gdp_", DEM_scen)
```

Comment:
Warum muss hier ein gdp davor?

Reply:
Aber müsste das SSP_scen dann da nicht auch noch mit drin stehen? Also gdp_SSP2_NAV_act?
Oder wollen wir das SSP scenario sowieso in das DemScen mit reinschreiben?